tests/requests_spec.lua

```diff
@@ -713,4 +713,23 @@ describe("Lsp request", function()
       assert.is.same(result.command.title, "references: 2")
     end
   end)
+
+  it("should return correct response for " .. custom_methods.JsxClosingTag, function()
+    utils.open_file "src/jsx_close_tag.ts"
```

Comment:
You opening not existing file this is why it isn't working:
```suggestion
    utils.open_file "src/jsx_close_tag.tsx"
```